packages/frontend/src/deviceMessages.ts

```diff
@@ -26,19 +26,16 @@ export async function updateDeviceChat(
     }
   }
 
-  await addDeviceMessage('changelog-version-1.46.0-version1', {
-    text: `What's new in 1.46.0?
+  await addDeviceMessage('changelog-version-1.50.0', {
+    text: `What's new in 1.50.0?
 
-🐣 New Onboarding: ⚡️ Sign up to secure fast chatmail servers (https://delta.chat/chatmail) or use classic e-mail servers
+🐣 Many accessibility improvements for keyboard navigation
+📋 Select and send multiple files at once
+📌 Webxdc apps can send notifications now
+🚀 Enable Webxdc realtime by default
+✨ Lots of UI improvements and bug fixes
```

Comment:
we do not want to say "Webxdc" in a user-facing way.

just a suggestion - i am not saying that every line is better :) - feel free to pick what you like and discard the remainder :)

```suggestion
🐣 Improved keyboard navigation
📋 Select and send multiple files at once
🎮 Enhanced in-chat apps: Get notifications and open supporting apps in context, i.e. open an added calendar entry directly
🚀 In-chat apps can be really fast now by the "realtime" support

MORE ✨ improvements and 🐜 bug fixes at https://get.delta.chat/#changelogs
```